argoexec/rockcraft.yaml

```diff
@@ -35,26 +34,14 @@ parts:
       - netbase
       - tzdata
 
-  base-snaps:
-    plugin: nil
-    stage-snaps:
-      - docker
-      - kubectl/1.24/stable
-    organize:
-      docker: bin/docker
-      kubectl: bin/kubectl
-    stage:
-      - bin/docker
-      - bin/kubectl
-
   builder:
     plugin: make
     after: [base-deps]
     source: https://github.com/argoproj/argo-workflows
     source-type: git
-    source-tag: v3.3.10
+    source-tag: v3.4.16
     build-snaps:
-      - go/1.17/stable
+      - go/1.20/stable
     build-packages:
```

Comment:
Do we really ned all of these ? the Dockerfile uses only these https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L8

Reply:
you are right, removing